change/vanilla-js-node-test-app-2020-10-13-17-19-12-node-e2e-tests.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "none",
+  "comment": "Add end-to-end testing framework over msal-node samples (#2414)",
+  "packageName": "vanilla-js-node-test-app",
```

Comment:
You should copy the `.beachballrc` file from another sample into this directory to ignore this from beachball